metpy/calc/indices.py

```diff
@@ -305,3 +305,63 @@ def significant_tornado(sbcape, surface_based_lcl_height, storm_helicity_1km, sh
     return ((sbcape / (1500. * units('J/kg'))) *
             surface_based_lcl_height *
             (storm_helicity_1km / (150. * units('m^2/s^2'))) * shear_6km)
+
+
+@exporter.export
+@check_units('[pressure]', '[speed]', '[speed]', '[length]', '[speed]', '[speed]')
+def critical_angle(pressure, u, v, heights, stormu, stormv):
+    r"""Calculate the critical angle.
+
+    The critical angle is the angle between the 10m storm-relative inflow vector
+    and the 10m-500m shear vector. A critical angle near 90 degrees indicates
+    that a storm in this environment on the indicated storm motion vector
+    is likely ingesting purely streamwise vorticity into its updraft, and Esterheld and
+    Giuliano (2008) showed that significantly tornadic supercells tend to occur in environments
+    with critical angles near 90 degrees.
+
+    Parameters
+    ----------
+    pressure : `pint.Quantity`
+        Pressures from sounding.
+    u : `pint.Quantity`
+        U-component of sounding winds.
+    v : `pint.Quantity`
+        V-component of sounding winds.
+    heights : `pint.Quantity`
+        Heights from sounding.
+    stormu : `pint.Quantity`
+        U-component of storm motion.
+    stormv : `pint.Quantity`
+        V-component of storm motion.
+
+    Returns
+    -------
+    `pint.Quantity'
```

Comment:
Closing here is a single quote instead of a backtick, which is angering the sphinx gods.

Reply:
I hope the sphinx has accepted the sacrifice of replacing a quote with a backtick lol.